Libraries/libutil/Sources/Relative.cpp

```diff
@@ -106,7 +140,24 @@ normalized() const
         output += (Traits::IsSeparator(_raw[n]) ? Traits::Separator : _raw[n]);
     }
 
-    output += NormalizePath(_raw, _raw.begin() + start, _raw.end(), absolute, true, Traits::Separator, &Traits::IsSeparator);
+    std::vector<std::string> relativeOutput = NormalizePathComponents(
+        _raw,
+        _raw.begin() + start,
+        _raw.end(),
+        absolute,
+        true,
+        Traits::Separator,
+        &Traits::IsSeparator
+    );
+
+    auto begin = relativeOutput.begin();
+    auto end = relativeOutput.end();
+    for (auto it = begin; it != end; ++it) {
+        output += *it;
+        if (std::next(it) != end) {
+          output += Traits::Separator;
+        }
+    }
```

Comment:
Put this loop into its own function? (Maybe that function already exists?)

Reply:
I don't think it does.  I'll check.